content/company/values.md

```diff
@@ -14,9 +14,15 @@ These values are the beliefs and principles that help us achieve our [goals](goa
 
 ## Values in detail
 
-### Be customer driven
+### Customer-first
 
-You proactively work on the right things by continuously orienting your goals around delivering value to your customers.
+You earn and keep the trust of our customers by putting their interests first, ahead of Sourcegraph's interests and your team's interests. Over the long term, we need to be a customer-funded business, and you'll help us get there by being customer-first.
+
+- We are not engineering-driven. We are not sales-driven. Being customer-first unites us all (engineering, sales, and every other team).
```

Comment:
```suggestion
- We are not engineering-first. We are not sales-first. Being customer-first unites us all (engineering, sales, and every other team).
```
I think aligning these gives them a stronger relation to "customer-first," and helps the idea resonate more. Just a thought!

Reply:
@CristinaBirkel Yes, this is a good improvement. Thank you!